pkg/virt-controller/watch/migration.go

```diff
@@ -1576,10 +1576,10 @@ func prepareNodeSelectorForHostCpuModel(node *k8sv1.Node, pod *k8sv1.Pod, source
 			return fmt.Errorf("node does not contain labal \"%s\" with information about host cpu model", virtv1.HostModelCPULabel)
 		}
 
-		hostCpuModelLabelKey = virtv1.HostModelCPULabel + hostCpuModel
-		pod.Spec.NodeSelector[hostCpuModelLabelKey] = hostModelLabelValue
+		nodeSelectorKeyForHostModel = virtv1.CPUModelLabel + hostCpuModel
+		pod.Spec.NodeSelector[nodeSelectorKeyForHostModel] = hostModelLabelValue
 
-		log.Log.Object(pod).Infof("cpu model label selector (\"%s\") defined for migration target pod", hostCpuModelLabelKey)
+		log.Log.Object(pod).Infof("host model label selector (\"%s\") defined for migration target pod", nodeSelectorKeyForHostModel)
```

Comment:
I think that actually the previous description is more accurate: `cpu model label selector (\"%s\") defined for...`